### PR TITLE
docs(json-schema): fix link to container sizes

### PR DESCRIPTION
**Pull request by revolunet**
fix broken link

### CODE REVIEW

File: _scalingo_json_schema/index.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ and the value should have the following structure:
 | field    | type    | description                                                            |
 | -------- | ------- | ---------------------------------------------------------------------- |
 | amount   | integer | Number of containers to start once the application is deployed         |
-| size     | string  | Container size `name` ([get the list of sizes](/container-sizes.html)) |
+| size     | string  | Container size `name` ([get the list of sizes](https://doc.scalingo.com/platform/internals/container-sizes)) |
```
Comment (EtienneM) [inline _scalingo_json_schema/index.md:183]: We actually want to target the documentation page explaining the API endpoint to list container sizes.

```suggestion
| size     | string  | Container size `name` ([get the list of sizes](/container_sizes#list-the-container-sizes-available)) |
```

**Review by EtienneM — CHANGES_REQUESTED** [review]
Thanks a lot

**Review by EtienneM — APPROVED** [review]

**Comment by revolunet** [comment]
Would you be interested in a PR with the working JSON-schema ?

With it you can autodetect `scalingo.json` files and provide linting and documentation right from the editor.

Capture d’écran 2024-02-06 à 10 46 09

---

Capture d’écran 2024-02-06 à 10 46 16

---

Capture d’écran 2024-02-06 à 10 49 02

**Comment by leo-scalingo** [comment]
Hi @revolunet 

Indeed it's something which would be really useful. Could you tell us a bit more about how you've generated/used this schema?

Thanks a lot

**Comment by revolunet** [comment]
More here : https://github.com/Scalingo/developers-documentation/pull/390

**Pull request merged**